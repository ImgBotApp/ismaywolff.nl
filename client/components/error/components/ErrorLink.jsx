import styled from 'styled-components'
import { colors } from '../../../styles'

const ErrorLink = styled.a`
  color: ${colors.darkRed};
`

export default ErrorLink

