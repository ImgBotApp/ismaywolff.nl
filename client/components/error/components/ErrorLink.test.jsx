import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import ErrorLink from './ErrorLink'

describe('<ErrorLink />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<ErrorLink />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
