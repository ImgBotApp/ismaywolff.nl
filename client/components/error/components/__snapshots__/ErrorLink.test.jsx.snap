// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorLink /> renders correctly 1`] = `
<a
  className="fvEVxn"
/>
`;
